app/models/transfer/between_plates.rb

```diff
@@ -83,7 +83,12 @@ def calculate_location_submissions # rubocop:todo Metrics/AbcSize
     destination_sources.each_with_object({}) do |dest_source, store|
       dest_loc, sources = *dest_source
 
-      found_pre_cap_groups = pre_cap_groups.select { |_uuid, group_details| group_details[:wells].sort == sources.sort }
+      # Instead of requiring an exact match between the sources from transfers
+      # and the pre-cap group wells, we check if the sources are a subset of
+      # the pre-cap group. This allows for failed wells that are not part of
+      # the transfers to be ignored.
+      found_pre_cap_groups =
```

Comment:
maybe show an example in the comments of whats in the sources.sort and group_details[:wells].sort? Are they just lists (arrays? hashes?) of well coordinate strings ('A1', 'B1' etc.)?